### DISCUSSION TITLE
Try my Python code in GoogleColab

### DISCUSSION THREAD

**Discussion by zyang91** [Geography-and-Planning/GEOG-510#8] [Announcements]
In all my Jupyter notebook, I insert a link that enable you open the notebook and try it in Google Colab! 
Feel free to try it, if you are interested

**Comment by Repjas02** [comment]
where is the link i will like to try it

**Reply by zyang91** [reply]
The link is always at the top of the document. (Blue button).But, if the notebook haven't finished updated, their will be no link. After finishing, the link will appear.

**Reply by zyang91** [reply]
![image](https://colab.research.google.com/assets/colab-badge.svg)
Look for this button

**Comment by zyang91** [comment]
If you have further questions, Please let me know!

Best regards,
Zhanchao

**Reply by Repjas02** [reply]
thank you can you help write this code too its my work at school

**Reply by zyang91** [reply]
Sorry. I don't help anyone write code. It's just my interests. 
If you need additional resources about python, please refer to [Python official website](https://www.python.org/).

**Reply by zyang91** [reply]
Also, many AI platform like ChatGPT are really great for your references! Maybe they can help you write some codes

**Reply by Repjas02** [reply]
Yeah I have a code I am practising I want to load it here so that you can as well make  your contribution to it I will be glad if you will be  wiling to help .

**Reply by Repjas02** [reply]
Oh really thanks for your useful insights it was hepful
